pkg/forklift-api/webhooks/mutating-webhook/mutators/ovirt-cert-mutator.go

```diff
@@ -69,9 +69,17 @@ func (mutator *OvirtCertMutator) Mutate(ar *admissionv1.AdmissionReview) *admiss
 			return util.ToAdmissionResponseError(err)
 		}
 
+		// Remove all new lines from both certs for the comparison.
+		secretCa := bytes.ReplaceAll(secret.Data["cacert"], []byte{0x0a}, []byte{})
+		retrivedCa := bytes.ReplaceAll(b, []byte{0x0a}, []byte{})
 		//check if the CA included in the secrete provided by the user and update it if needed
-		if !strings.Contains(string(secret.Data["cacert"]), string(b)) {
-			secret.Data["cacert"] = append(secret.Data["cacert"], b...)
+		if !bytes.Contains(secretCa, retrivedCa) {
+			if bytes.HasSuffix(secretCa, []byte{0x0a}) {
+				secret.Data["cacert"] = append(secret.Data["cacert"], b...)
+			} else {
+				newCaVal := append(secret.Data["cacert"], 0x0a)
+				secret.Data["cacert"] = append(newCaVal, b...)
+			}
```

Comment:
how about adding unit tests for the cases that are now covered by this?

Reply:
good idea, will add it soon in a separate PR.

Reply:
ack